guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -1,24 +1,25 @@
 [id="Renaming_Smart_Proxy_{context}"]
 = Renaming {SmartProxyServerTitle}
 
-The host name of {SmartProxyServer} is referenced by {ProjectServer} components, and all hosts registered to it.
-This procedure ensures that you update all references to the new host name.
+The host name of {SmartProxyServer} is referenced by {ProjectServer} components and all hosts registered to it.
+This procedure ensures that in addition to renaming {SmartProxyServer}, you also update all references to the new host name.
 
-[NOTE]
+[WARNING]
 ====
-* Both the `hostname` and `hostname -f` commands must return the FQDN of {SmartProxyServer} or the `{project-change-hostname}` script will fail to complete.
-
-* If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _old_fqdn_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
+Renaming your {SmartProxyServer} host shuts down all {ProjectServer} services on that host.
+The services restart after the renaming is complete.
 ====
 
 .Prerequisites
-* Backup {SmartProxyServer}.
-The `{project-change-hostname}` script makes irreversible changes to {SmartProxyServer}.
-If the renaming process is not successful, you must restore it from a backup.
-+
-Perform a backup before changing a host name.
+* Back up your {SmartProxyServer} before renaming.
+If you fail to successfully rename it, restore it from the backup.
 For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_{context}[].
 
+* Run the `hostname` and `hostname -f` commands on {SmartProxyServer}.
+Both commands must return the FQDN of {SmartProxyServer} or the `{project-change-hostname}` script will fail to complete.
++
+If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _old_fqdn_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
```

Comment:
```suggestion
If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _My_Old_FQDN_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
```